deploy/kiali/kiali_cr.yaml

```diff
@@ -584,7 +598,9 @@ spec:
 #   use_kiali_token: When true and if auth.type is "bearer", the same OAuth token used for authentication in Kiali will be used for the API calls to Jaeger Query,
 #       and auth.token config is ignored then.
 #   username: Username to be used when making requests to Jaeger, for basic authentication. User only requires viewer permissions.
-# is_core_component: Used in the Components health feature. When true, the unhealthy scenarios will be raised as errors. Otherwise, they will be raised as a warning.
+# component_status: Used in the Components health feature.
+#   is_core: When true, the unhealthy scenarios will be raised as errors. Otherwise, they will be raised as a warning.
+#   health_check_url: The url which kiali will ping to determine whether the component is reachable or not. It defaults to `in_cluster_url` when not provided.
```

Comment:
all peer settings should be sorted alphabetically. "h" before "i" :)

```suggestion
#   health_check_url: The url which kiali will ping to determine whether the component is reachable or not. It defaults to `in_cluster_url` when not provided.
#   is_core: When true, the unhealthy scenarios will be raised as errors. Otherwise, they will be raised as a warning.
```